snapshots/animal_welfare/2023-09-01/chick_culling_laws.csv.dvc

```diff
@@ -0,0 +1,129 @@
+meta:
+  origin:
+    dataset_title_owid: Chick culling laws
+    dataset_description_owid: Male baby chicks are commonly killed in laying hens
+      hatcheries all around the world, as they provide no value for the egg production
+      industry. Only a minority of country has laws to ban this practice. This dataset
+      is a compilation of evidence (from official institutions and in some cases from
+      the news) of chick culling taking place in a country, and, in some cases, of
+      existing laws banning it.
+    producer: Various sources
+    citation_producer: >-
```

Comment:
Yes, I think this field should be called `citation_full` (or something along those lines) because often the producer doesn't specify how to cite them, but we always need a full citation.
In this particular case, the producer is "Various sources", and the full citation should list all those sources. I'll add something about this case in the guidelines.